packages/sdk-react/src/ui/FileViewer/FileViewer.tsx

```diff
@@ -152,143 +68,60 @@ const ErrorComponent = ({
 };
 
 const FileViewerComponent = ({
-  url,
```

Comment:
It looks like `url` is the important thing here, if it is empty, we probably should not render the component.